### ISSUE TITLE
[Feature Request] Allow `-` for stdin

### ISSUE THREAD

**Issue by oganexon** [c4spar/deno-cliffy#99]
Using - as a filename to mean stdin.
```sh
myCommand --someOptions -
```

```ts
new Command()
  .name("myCommand")
  .option("--someOptions", "desc")
  .option("-", "stdin")
  .action((options: Options) => console.log(options.stdin))
  // or using a Symbol
  .action((options: Options) => console.log(options[cliffy.stdin]))
```
Like unix commands, such as :
```sh
cat -
```

**Comment by c4spar** [comment]
Hi @oganexon,

What about using a normal option for specifying to read from stdin?
```typescript
await new Command()
  .option("-i, --stdin", "Read from stdin.")
  .action(async ({ stdin }) => {
    if (stdin) {
      await Deno.copy(Deno.stdin, Deno.stdout);
    }
  })
  .parse();
```

I'm not shure if we should implement it as an option. What about passing the `-` as an argument to the action handler?
```typescript
await new Command()
  .argument("<input:string>", "Input file or - for reading from stdin.")
  .action(async (_, input: string) => {
    if (input === "-") {
      await Deno.copy(Deno.stdin, Deno.stdout);
    }
  })
  .parse();
```

**Comment by oganexon** [comment]
> ```ts
> await new Command()
>   .argument("<input:string>", "Input file or - for reading from stdin.")
>   .action(async (_, input: string) => {
>     if (input === "-") {
>       await Deno.copy(Deno.stdin, Deno.stdout);
>     }
>   })
>   .parse();
> ```

@c4spar This solution is just what I was looking for, thank you!

**Comment by c4spar** [comment]
Hey @oganexon,

This solution was not yet working but it's fixed in v0.14.3 now.
I had a mistake in my previous example but here is a working one:
```typescript
    await new Command()
      .arguments("<input:string>")
      .action(async (_, input: string) => {
        if (input === "-") {
          await Deno.copy(Deno.stdin, Deno.stdout);
        }
      })
      .parse();
```
